### ISSUE TITLE
the difference between node and vertex

### ISSUE THREAD

**Issue by tanjia123456** [PetarV-/GAT#50]
I'm sorry to bother you
could you please examplain the the difference between node and vertex? My datasets is mindboggle(brain) contain the vertex. Many code are use node, So I feel very confused.

**Comment by gcucurull** [comment]
Hello,

there isn't really a difference between a node and a vertex. Some people use the word "node" and others use the word "vertex", but both have the same meaning.

**Comment by tanjia123456** [comment]
> 
> 
> Hello,
> 
> there isn't really a difference between a node and a vertex. Some people use the word "node" and others use the word "vertex", but both have the same meaning.

but my vertex is 3d coordinate, there is no matter?

**Comment by Zer0Credibility** [comment]
you can encode spacial dimension coordinates as node feature vector elements if you'd like to keep that information.

**Comment by tanjia123456** [comment]
> 
> 
> you can encode spacial dimension coordinates as node feature vector elements if you'd like to keep that information.
hi, thank you for your reply.   how to achieve it? I was going to directly bring the 3D coordinates into.
In addition, what is the storage format of these data?

**Comment by tanjia123456** [comment]
@gcucurull 
Hello, every person's brain can be represented by a graph, and there are a total of 80 people's data(80 subjects). As far as I know, the original GAT is to divide the data set of nodes, but here I want to divide the data set according to the subject. May I divide the first 70 subjects into the training set and the last 10 into the test set, so how do I use GAT for training? I want to use GAT for image segmentation.
Please take the time to guide me the following, it is greatly appreciated.
